src/transformers/models/llava/modeling_llava.py

```diff
@@ -92,12 +92,19 @@ class LlavaCausalLMOutputWithPast(ModelOutput):
 class LlavaMultiModalProjector(nn.Module):
     def __init__(self, config: LlavaConfig):
         super().__init__()
+        if config.projector_layernorm:
```

Comment:
not really transformers philosophy :| 
Was thinking that we could always add a layer norm, but just set the parameters to make it perform identity for the other checkpoints? (kind of a hack but might stabilize training / fine tuning ?)

Reply:
Agree, Sylvain would have never allowed this so let's keep it that way.

New paper = new model = no new config attributes but rather a separate model with copied from. Although I know that philosophy is sometimes a bit extreme, but it has always worked out in the past